### PR TITLE
[OPS-3651] Export updated events_event feature, as the revert is what…

### PR DESCRIPTION
**Pull request by cafuego**
… causes S-API index reset.

### CODE REVIEW

File: html/sites/all/modules/custom/events_event/events_event.features.field_instance.inc [hunk 1]
```diff
@@ -112,7 +112,7 @@ function events_event_field_default_field_instances() {
     'bundle' => 'ev_event',
     'default_value' => array(
       0 => array(
-        'tid' => 5576,
+        'tid' => 44528,
```
Comment (attiks) [inline html/sites/all/modules/custom/events_event/events_event.features.field_instance.inc]: can you remove this from the PR

File: html/sites/all/modules/custom/events_event/events_event.features.taxonomy.inc [hunk 2]
```diff
@@ -53,7 +53,7 @@ function events_event_taxonomy_default_vocabularies() {
       'name' => 'Location',
       'machine_name' => 'ev_location',
       'description' => '',
-      'hierarchy' => 1,
+      'hierarchy' => 0,
```
Comment (attiks) [inline html/sites/all/modules/custom/events_event/events_event.features.taxonomy.inc]: This should stay

File: html/sites/all/modules/custom/events_event/events_event.info [hunk 3]
```diff
@@ -2,7 +2,7 @@ name = Event
 description = Event
 core = 7.x
 package = Events
-version = 7.x-1.0
+version = 7.x-1.1
```
Comment (attiks) [inline html/sites/all/modules/custom/events_event/events_event.info]: Leave the version as is please

File: html/sites/all/modules/custom/events_event/events_event.module [hunk 4]
```diff
@@ -1186,22 +1186,21 @@ function events_event_clear_term_cache($term) {
  * Get locations of an event as string.
  */
 function events_event_get_event_locations($node) {
-  $output = '';
+  $output = [];
```
Comment (attiks) [inline html/sites/all/modules/custom/events_event/events_event.module]: can you use array() to be consistent with other code

**Comment by attiks** [comment]
@cafuego will merge fixes in other PR

**Comment by attiks** [comment]
See https://github.com/UN-OCHA/events-site/pull/200

**Comment by cafuego** [comment]
The fix for the line 1201 warning dropped on the floor, though. See https://github.com/UN-OCHA/events-site/pull/204

**Pull request closed without merging**